docs/source/develop.rst

```diff
@@ -79,17 +79,24 @@ Make a fork of the main `Dask repository <https://github.com/dask/dask>`_ and
 clone the fork::
 
    git clone https://github.com/<your-github-username>/dask
+   cd dask
+
+You should also pull the latest tags (otherwise the pip dependency resolver can
+sometimes get stuck installing Dask)::
```

Comment:
In general I try to frame things with a positive emphasis. For example, "ensures `pip` works correctly" as opposed to "makes sure `pip` doesn't crash"

```suggestion
You should also pull the latest git tags (this ensures ``pip``'s dependency resolver
can successfully install Dask)::
```